pages/design/rule-design.md

```diff
@@ -2,35 +2,73 @@
+
+### Rule name
+
+The agreed conventions for naming of rules are as follows:
+
+- Be succinct, direct and declarative, and avoid unnecessary words, _e.g. an, the_.
+- Use declarative verbs, _e.g. has, have, is_.
+- Use back ticks to indicate words from code, _e.g. `label` v. Label_.
+- Identify code as an element or attribute as appropriate, _e.g. `title` element, `title` attribute_.
+- Use sentence case, _e.g. "Heading is descriptive"_, unless using code, _e.g. "`id` attribute value is unique"_, or initials, _e.g. "HTML page has title"_. Note: names starting with a backtick also need to be wrapped with quotes for the parser. 
+- Use singular tense so that necessary plurals stand out, _e.g. "`video` element has captions"_.
+- Front load the applicable thing, _e.g. "Button has accessible name"_.
+- Refer to each applicable thing in a consistent way across rules, especially when related or checking the same applicable thing.
+- Describe the passing condition, _e.g. "`id` attribute value is unique", rather than "page has no duplicate `id`s"_.
+- Do not use hyphens or dashes unless correct for code, _e.g. `aria-*`_.
+- Do not use camelCase unless correct for code, _e.g. `iframe` not iFrame, `feImage` in SVG_.
+- If unsure, refer to WCAG language used.
+- Aim to be unique, which should happen if title includes the applicable thing and passing condition.
 
-The rule must have a unique name, preferably a two or three word topic, as well as an identifier. This uses the following format: **SC#-#-#-identifier**
+### Rule type
 
-- **SC#-#-#**: This is an identifier for the criterion to which the test case applies. #-#-# stands for the number of that criterion, such as SC4-1-2.
+The rule type is either atomic or composite.
 
-- **+SC#-#-#**: This can be used if the test case applies to multiple success criteria, such as SC1-1-1+SC4-1-2-identifier. The numbers are in the same order as they are used in WCAG.
+For more details, see [ACT Rules Format: Rule Types](https://www.w3.org/TR/act-rules-format/#rule-types).
 
-- **identifier**: This must be a lower case identifier of the test, preferably no more then 3 words. It can only contain alphanumeric values or a dash (-).
+### Description
 
-### Test aspects
+A brief description of what the rule does. This should be declarative rather than negative. It should begin with "This rule checks that..." or similar. It should use plain language, and be a well-formed grammatically correct sentence that finishes with a full stop/period. 
 
-A test aspect is a part of a test subject that must be available in order to properly run the test. The ACT Rules Format defines the following:
+> _For example: "This rule checks that each ARIA state or property has a valid value."_
+
+### Accessibility requirements
+
+A list of all accessibility requirements that are not satisfied if one or more outcomes of a rule fails. This could be one or more WCAG success criteria, or a WAI-ARIA rule, for example.
```

Comment:
```suggestion
A list of all accessibility requirements that are not satisfied if one or more outcomes of a rule fails. This could be one or more WCAG success criteria, or a WAI-ARIA rule, for example.

**Note**: When linking to WCAG or other sources that may have several versions, use the version number where the relevant requirement was first introduced.
```